src/Orleans.Core/Configuration/Options/ClusterMembershipOptions.cs

```diff
@@ -115,7 +101,7 @@ public class ClusterMembershipOptions
         public TimeSpan? DefunctSiloCleanupPeriod { get; set; } = TimeSpan.FromHours(1);
 
         /// <summary>
-        /// Gets the period after which a silo is ignored for initial connectivity validation if it has not updated its heartbeat in the silo membership table.
+        /// Gets the period after which a silo that didn't updated its status in the table causes a warning to be logged.
```

Comment:
The comment implies this logic is changing, but I don't think we should change this logic, should we?

Reply:
This property was used for both cases (ignore for initial connectivity and print a warning on table refresh). Now we only print the warning.

The second use case wasn't in the comment.